nlp/README.md

```diff
@@ -0,0 +1,11 @@
+# ML6 NLP Quick Tips
+
+Current content:
+
+- TOTW 01 (21/01/2021) - _Multilingual Sentence Embeddings_:
```

Comment:
Will we call them TOTW or QT? The former implies a regular schedule which we may not be able to deliver?

Reply:
Good one, might be easier to avoid superfluous wording and just use?
`[titel](datum): description`